README.rst

```diff
@@ -39,7 +39,7 @@ You can install from source by cloning this repository and issuing a pip install
 
     git clone https://github.com/aws/sagemaker-python-sdk.git
     python setup.py sdist
-    pip install dist/sagemaker-1.0.3.tar.gz
```

Comment:
If you want to bump the version here, please update setup.py and CHANGELOG

Reply:
Fixed! Thanks